sdcm/results_analyze/test.py

```diff
@@ -226,6 +226,7 @@ class SoftwareVersions(ClassBase):
     scylla_enterprise_server: ScyllaEnterpriseVersionInfo = None
 
     @property
+    # pylint: disable=unsubscriptable-object
```

Comment:
why did you add it here? it shouldn't be subscriptable object

Reply:
pylint version I had didn't support proper linting for union type hints, fixed and removed that line